components/raftstore-v2/src/router/message.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright 2016 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
2022

Reply:
ASF suggests using the interception year of the project consistently for all its files. But of course it's not a requirement.

* https://github.com/apache/skywalking/issues/10223
* https://issues.apache.org/jira/browse/LEGAL-629